htdocs/societe/list.php

```diff
@@ -836,7 +836,7 @@
 if (! empty($arrayfields['s.status']['checked']))
 {
     print '<td class="liste_titre maxwidthonsmartphone" align="center">';
-    print $form->selectarray('search_status', array('0'=>$langs->trans('ActivityCeased'),'1'=>$langs->trans('InActivity')),$search_status);
+    print $form->selectarray('search_status', array('-1'=>'', '0'=>$langs->trans('ActivityCeased'),'1'=>$langs->trans('InActivity')),$search_status);
```

Comment:
To introduce a "nothing selected" among possible values into a combo, it is better to add the parameter "empty" to 1 into the call of function selectarray.